packages/ios_platform_images/pigeons/platform_images.dart

```diff
@@ -0,0 +1,39 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:pigeon/pigeon.dart';
+
+@ConfigurePigeon(PigeonOptions(
+  dartOut: 'lib/platform_images_api.g.dart',
+  objcHeaderOut: 'ios/Classes/PlatformImagesApi.g.h',
+  objcSourceOut: 'ios/Classes/PlatformImagesApi.g.m',
+  objcOptions: ObjcOptions(
+    prefix: 'FLT',
+  ),
+  copyrightHeader: 'pigeons/copyright.txt',
+))
+class PlatformImage {
+  double? scale;
+  Uint8List? bytes;
```

Comment:
Why are these nullable? What does it mean for an image to have a null scale or null bytes?

Reply:
Yeah that doesn't make sense, I think I just did it without thinking to get rid of compiler errors.

Should I give them default values like the code below or add a `late` modifier?
```dart
double scale = 1;
Uint8List bytes = Uint8List(0);
```

Reply:
You can define a Dart constructor for them with required parameters.